tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -388,6 +391,133 @@ test.describe( 'Set up accounts', () => {
 		} );
 	} );
 
+	test.describe( 'Google Ads card', () => {
+		test.beforeAll( async () => {
+			// Mock Jetpack as connected
+			await setUpAccountsPage.mockJetpackConnected(
+				'Test user',
+				'jetpack@example.com'
+			);
+
+			await setUpAccountsPage.fulfillMCAccounts( [
```

Comment:
```suggestion
			await setUpAccountsPage.mockMCHasAccounts()
```
@ankitrox Can we replace with the mock function we already have?

Reply:
@ankitrox Just checking you saw my comment. thanks!

Reply:
Sorry, missed that one. :)

I have updated now to use `mockMCHasAccounts`.

Thanks